### ISSUE TITLE
Accessibility: Incorrect tab order for search options dropdown

### ISSUE THREAD

**Issue by GuySartorelli** [silverstripe/silverstripe-admin#2085]
When opening the "Search options" dropdown (e.g. in a gridfield or in /admin/pages) with keyboard, you then have to tab BACKWARDS to access the form.

<img width="526" height="483" alt="Image" src="https://github.com/user-attachments/assets/918079bf-560d-44f4-aee6-81787ec7aa91" />

The form should be AFTER the search options button in the tab order.

Optionally opening search options should also put your focus on the first item in the search options form.

## PRs

- https://github.com/silverstripe/silverstripe-admin/pull/2091
